lblod/frontend-lokaal-mandatenbeheer
# Lmb 941 | add warning if bcsd voorzitter is not schepen

## Description

When the selected voorzitter in BCSD does not have a mandaat schepen/toegevoegde schepen than we have to show a warning for it.

extra: As there already is a bcsdVoorzitter alert we will extends this one

## How to test

1. Add a bcsd voorzitter without a person => no alert
2. Assign a person that is not a schepen already => alert should be shown
3. Add a schepen mandaat for the voorzitter persoon => alert should be gone
4. Remove the schepen mandaat => alert should pop up again

## Attachments

![image](https://github.com/user-attachments/assets/6f36ba19-8c77-494e-b720-d2264c561dc6)
